client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -42,6 +43,13 @@ export class CommentNavigation extends Component {
 		status: 'unapproved',
 	};
 
+	bulkDeletePermanently = () => {
+		const { setBulkStatus, translate } = this.props;
+		if ( isUndefined( window ) || window.confirm( translate( 'Delete these comments permanently?' ) ) ) {
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Delete this comment permanently?' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/comments/bulk-actions-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).